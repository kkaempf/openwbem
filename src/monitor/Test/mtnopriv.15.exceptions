PrivilegeManagerException   CheckException: readLink: insufficient privileges
IPCIOException
