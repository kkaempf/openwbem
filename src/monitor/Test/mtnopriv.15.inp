readLink /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/link-exists
readLink /montest-689acb0e1ec89f45-7085a2a1780f5f42/test_dir/link-doesnt-exist
 
