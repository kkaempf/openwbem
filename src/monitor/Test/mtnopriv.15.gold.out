User: owprovdr

readLink
Exception:
  type: PrivilegeManagerException
  msg:  CheckException: readLink: insufficient privileges

readLink
Exception:
  type: IPCIOException
